src/NuGet.Server.V2.OWinSampleHost/Controllers/NuGetPrivateODataController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NuGet.Server.V2.Controllers;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+namespace NuGet.Server.V2.OWinSampleHost
+{
+    [Authorize]
+    //Requires user to be authorized to use this feed.
+    public class NuGetPrivateODataController : NuGetODataController
```

Comment:
I thought you wanted the New NuGet.Server to be IIS-hosted?
My idea was to keep this project as a sample in source-form only, demonstrating the usage of the NuGet.Server.V2 package in a selfhosted-scenario. 
It that remains the case, i should maybe rename the project to NuGet.Server.V2.Samples.Owinhost or something else to clearly indicate that it is a sample project. And maybe put it in a own 'samples' folder?